overlord/servicestate/quota_control.go

```diff
@@ -44,6 +52,54 @@ func init() {
 	checkSystemdVersion()
 }
 
+func checkMemoryCGroupEnabled() {
```

Comment:
Is this function actually called outside the tests? If not please ensure that we have a test that catches this.

Reply:
It's called during the init() but seems like a merge issue after separating it to enable file injection during tests